#CU-2ynbwg5 #CU-31gy38v #xcm  #xcmp security and testing hardening (#2118)
ComposableFi/composable: docs/docs/internal/xcmp/xcm-examples.md

List of useful to have XCM messages working for testing and playbooks.

> For general flow on how to make XCM and transfer happen, look [here](./ping.plantuml)

Reviewer: ```suggestion
 For details on the general workflow of XCM and transfers [look here](./ping.plantuml)
```
Nit expression
